### DISCUSSION TITLE
Is there anything needed to call, to create the desktop entries?

### DISCUSSION THREAD

**Discussion by ghost** [mrshmllow/affinity-nix#27] [Q&A] [answered]
Followed instructions on the flake.nix and added the packages to systempackages. Now how do I make use of [desktopItems.nix](https://github.com/mrshmllow/affinity-nix/blob/main/desktopItems.nix)?

**Comment by mrshmllow** [comment, marked as answer]
Installing the packages in any of the 3 ways outlined in the readme will add the desktop entries. There are going to be 2 because wine also adds some, which is super annoying

**Reply by ghost** [reply]
It might have just needed a restart or another configuration switch, they are there now
